src/App.js

```diff
@@ -1,25 +1,46 @@
-import logo from './logo.svg';
 import './App.css';
+import { Component, createElement } from 'react';
+import Header from './components/Header'
+import Module from './components/Module';
+import { getModule1Content } from './content';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends Component {
+    render() {
+        return createElement('div', { className: 'App' },
```

Comment:
let's agree, no longer use this way of creating React components.
For simplicity we have a powerful tags-like <> declarations instead of components factories.
Also almost 99% modern React apps and code-bases using functional approach instead of Class-like.
There are many reasons for that: 
- cleaner. more readable  code, no "noisy" declarations/definitions
- simpler for testing, straight functional in-out approach
- better inter-component composition
- no harming `this` context
- clear, simple and understandable closures
- ability to use hooks
- etc...

I would strongly recommend to migrate your app for standard way, how React app are creating